src/templates/components/provenance.ejs

```diff
@@ -27,12 +27,14 @@
 		const withholdings = provenance.withholdings || provenance.oiaRequest?.withholdings;
 
 		const [dateType, date] = (() => {
-			if (provenance.published) {
+			if (provenance.published && provenance.method !== 'Released under the OIA') {
 				return ['Published', provenance.published];
 			} else if (provenance.extracted && !locals.options?.omitExtractedDate) {
 				return ['Extracted', provenance.extracted];
 			} else if (provenance.released) {
 				return ['Released', provenance.released];
+			} else if (provenance.published && provenance.method === 'Released under the OIA') {
+				return ['Published', provenance.published];
```

Comment:
Isn't this added branch doing the same thing as modified? I guess this is this just meant to be a catch for if something has a method where it was released under the OIA, but somehow it doesn't have the released field populated?

Reply:
Because I amended line 30 to not apply when there is a published date and the provenance method is "Released under the OIA", I need to have an extra case where that is true so there won't be any edge case where nothing shows. But it needs to have lower priority, since provenance for information released under the OIA should prefer the released date over a published date

Reply:
Got it, so that matches what I suspected